std/digest/murmurhash.d

```diff
@@ -0,0 +1,751 @@
+/**
+Computes $(LINK2 https://en.wikipedia.org/wiki/MurmurHash, MurmurHash) hashes
+of arbitrary data. MurmurHash is a non-cryptographic hash function suitable
+for general hash-based lookup. It is optimized for x86 but can be used on
+all architectures.
+
+The current version is MurmurHash3, which yields a 32-bit or 128-bit hash value.
+The older MurmurHash 1 and 2 are currently not supported.
```

Comment:
It just sounded like a "missing feature", it was the "currently"  that triggered my internal _this library_ is incomplete flag.
In any case - it's a tiny nitpick ;-)